vendor/github.com/stripe/goproxy/https.go

```diff
@@ -119,21 +120,21 @@ func (proxy *ProxyHttpServer) handleHttps(w http.ResponseWriter, r *http.Request
 			host += ":80"
 		}
 
-		var httpsProxyURL string = proxy.HttpsProxyAddr
+		var httpsProxyString string = proxy.HttpsProxyAddr
```

Comment:
OOC why change the var here? Just because it isn't an `url.URL`?

Reply:
Yep just trying to be clearer about the type! Confused myself on this a couple times